frame/examples/basic/src/benchmarking.rs

```diff
@@ -21,8 +21,8 @@
 #![cfg(feature = "runtime-benchmarks")]
 
 use crate::*;
-use frame_benchmarking::v2::*;
-use frame_system::RawOrigin;
+use frame::deps::frame_benchmarking::v2::*;
```

Comment:
would be good to have a benchmarking_prelude in frame crate now.

Reply:
Yes I agree